### PR TITLE
Added ability to appove posts, and get a list of those approved posts.

### PR DESCRIPTION
**Pull request by thequbit**
Note these are just end points, and are not yet hooked up on the admin client side.

### CODE REVIEW

**Comment by thequbit** [comment]
Approve Post:
https://github.com/hhroc/yellr-server/wiki/admin-approve_post.json

File: yellr-serv/yellrserv/client_views.py [hunk 1]
```diff
@@ -682,15 +683,15 @@ def upload_media(request):
...
 
-            media_extention="processing"
+            #media_extention="processing"
 
             # generate a unique file name to store the file to
-            unique = uuid.uuid4()
-            file_name = '{0}'.format(unique) #,media_extention)
-            file_path = os.path.join(system_config['upload_dir'], file_name)
+            unique = str(uuid.uuid4())
+            #file_name = '{0}'.format(unique) #,media_extention)
```
Comment (Nolski) [inline yellr-serv/yellrserv/client_views.py]: ?

File: yellr-serv/yellrserv/client_views.py [hunk 2]
```diff
@@ -804,12 +811,14 @@ def upload_media(request):
                 else:
                     error_text = 'invalid audio file'
                     raise Exception('')
+                '''
+                pass
 
             #I don't think the user has a way to upload files of this type besides typing in the box
             #so it doesn't need as robust detection.
             elif media_type == 'text':
```
Comment (Nolski) [inline yellr-serv/yellrserv/client_views.py]:
```python
if media_type == 'text':
    error_text = 'invalid media type'
    raise Exception('invalid media type')
```

**Comment by thequbit** [comment]
So the client views need to get the same love that the admin views just got on the last clean up (and docs). 

I'm in the middle of doing that,  and it looks like some of it got wrapped into this PR.

This PR passes all automated tests,  and all manual testa on the app. 

If you're worried,  I can try and roll back that part of the code and submit the approved post changes again.

**Comment by thequbit** [comment]
There is a big cleanup PR coming that will include this. Closing, as things may change.

**Pull request closed without merging**